docs/troubleshooting/bug_report.md

```diff
@@ -1,43 +1,43 @@
 # Bug Report
 
-When something isn't working as expected you should [open an issue](https://github.com/zwave-js/zwavejs2mqtt/issues/new/choose) on Github. To help us debugging your issue you should provide some additional informations like logs and node export if needed.
+When something isn't working as expected you should [open an issue](https://github.com/zwave-js/zwavejs2mqtt/issues/new/choose) on Github. To help us debug your issue you should provide all available information like logs and a node export, if needed.
 
 ## Export Logs
 
-The are two main loggers, one for zwavejs2mqtt and one for zwave-js module. To help us finding what's going on, when reporting a bug, you should attach the logs coming from both loggers.
+The are two main loggers, one for Zwavejs2Mqtt and one for the Z-Wave JS module. To help us find out what's going on, you should attach the logs coming from both loggers when reporting a bug.
 
-Both loggers options are configurable from the UI in `Settings` page.
+Both logger options are configured from the `Settings` page.
 
 ### Application logs
 
-Zwavejs2mqtt logger can be configured in `General` section, enable `Log enabled` switch and `Log To File` and set `Log Level` to `silly`
+Zwavejs2Mqtt logger can be configured in `General` section, enable `Log enabled` switch and `Log To File` and set `Log Level` to `Silly`
 
 > Log file name: `zwavejs2mqtt.log`
 
 ![Enable logging](../_images/log_z2m.gif)
 
 ### Driver logs
 
```

Comment:
Similarly, this should mention when driver logs are necessary (i.e. for everything that's not purely an UI issue).